### ISSUE TITLE
update design

### ISSUE THREAD

**Issue by dominguez-aureax** [dominguez-aureax/aureax-app#24]
Currently, I have new ideas of how the application will look after the sprint review.

If a user is not logged in there will primarily be two screens:
1. Login Screen
2. Sign Up Screen

If a user is signed in there can currently be 5 screens (at least):
1. The Panel (separate views for admins and non-admins)
2. Job Postings/Job Campaigns
  - Unsure if I'm going to Opt for a separated view or compact view here. I Will update once the design is complete
3. Active Campaign
4. Settings/User Information
  - Unsure of how many settings will be available to the User which is why it is compiled in the same section. (Think Robinhood and Chick-fil-A application as this idea was inspired from there).
  - I believe the basic referral link will also be here
5. Communication Screen
  - TBA. This could be messaging/article/forum style depending on feedback.
  
For these pages, there will be an app bar at the bottom of the screen to make it easier to access these screens

If a user is referred to the application there will be similar options: 
1. The User will be asked to sign a form before continuing to track referrals. They will be classified as 'guests'
2. Job Postings/Job Campaigns
  - Unsure if the share option will be disabled for guests
3. Active Campaign
4. Settings/User Information
5. Communication Sreen
  - This may be hidden for guests

**Comment by dominguez-aureax** [comment]
Updated Login Page 
![Login Screen](https://user-images.githubusercontent.com/85235510/123810656-94ff1e80-d8c0-11eb-8f23-06955f01088a.png)

**Comment by dominguez-aureax** [comment]
Updated Sign Up Screen
![Sign Up Screen](https://user-images.githubusercontent.com/85235510/123810841-b9f39180-d8c0-11eb-9618-edfa3ee07ef2.png)

**Comment by dominguez-aureax** [comment]
Updated Admin Panel
![Admin Panel](https://user-images.githubusercontent.com/85235510/123822194-10b19900-d8ca-11eb-83a1-7f4aeb827c30.png)

**Comment by dominguez-aureax** [comment]
Jobs Posting Page
![Jobs Page](https://user-images.githubusercontent.com/85235510/124015298-842ed580-d9b2-11eb-8846-cbe90ef903f2.png)

**Comment by dominguez-aureax** [comment]
Very basic drafts as I want to polish during the coding phase, the jobs will be scrollable and a search bar will be added

**Comment by dominguez-aureax** [comment]
Going to put the active campaign on hold and will work on settings when the options are figured out.

**Comment by dominguez-aureax** [comment]
@jsstallings let me know what you think of these rough designs. Plan on posting the Guest Form view sometime later today, or tomorrow but going to knock out some of the changes right now.

**Comment by jsstallings** [comment]
Love the look @dominguez-aureax.  I'll come back with some detailed comments based on your requirements post above, but absolutely look and feel is great, clean and simple.

**Comment by jsstallings** [comment]
And I think we need just the referral form page itself, which can be filled out as guest or as a non-admin user.  Unless I missed it above, is it the sign up form?  If so, all good, we can just call it that instead of sign up, or maybe sign up is fine.  We can discuss tomorrow.

**Comment by dominguez-aureax** [comment]
Yes I plan on having an add referral button under the page
Admin/User Panel > Referrals > Add Referral which will be similar to the page where the user has to sign through the referral link.
Not sure if that makes the best of sense before seeing the design, but I will work on having that ready tomorrow.

**Comment by dominguez-aureax** [comment]
Here is the view of adding a referral 
![Refferals](https://user-images.githubusercontent.com/85235510/124338567-a83b1400-db76-11eb-8fc2-f601046cd597.png)

**Comment by dominguez-aureax** [comment]
The screen will be similar when someone was referred to track their information and jobs interested.

**Comment by dominguez-aureax** [comment]
Navigating through the screens has caused an error with authentication. 
`he caller does not have permission to execute the specified operation.`
Looking into the matter atm

**Comment by dominguez-aureax** [comment]
`
E/flutter ( 6651): [ERROR:flutter/lib/ui/ui_dart_state.cc(199)] Unhandled Exception: dependOnInheritedWidgetOfExactType<_InheritedProviderScope<User?>>() or dependOnInheritedElement() was called before _AuthenticationWrapperState.initState() completed.
E/flutter ( 6651): When an inherited widget changes, for example if the value of Theme.of() changes, its dependent widgets are rebuilt. If the dependent widget's reference to the inherited widget is in a constructor or an initState() method, then the rebuilt dependent widget will not reflect the changes in the inherited widget.
E/flutter ( 6651): Typically references to inherited widgets should occur in widget build() methods. Alternatively, initialization based on inherited widgets can be placed in the didChangeDependencies method, which is called after initState and whenever the dependencies change thereafter.
`

**Comment by dominguez-aureax** [comment]
@jsstallings Currently having extreme difficulty with the UI. Apparently, it keeps collapsing on itself. Going to put this on hold and just get a website working. Extremely sorry for the delay.
